clarifai/client/model_client.py

```diff
@@ -204,6 +204,9 @@ def _define_functions(self):
 
             def bind_f(method_name, method_argnames, call_func, async_call_func):
                 def sync_f(*args, **kwargs):
+                    # Extract with_proto parameter if present
+                    with_proto = kwargs.pop('with_proto', False)
```

Comment:
can we ad "with_proto" as a constant and then in ModelClass verify that a user provided method does not have that argument name when they subclass ModelClass. That vailidation should be in method_signatures.py.